mungegithub/README.md

```diff
@@ -33,9 +34,17 @@ As the repo admin or the directory owner, one may approve and merge this PR into
 
   >  Note: Do not comment "/approve" or add "approved" label unless you are 100% sure you want this change, because after you say that, the pr will be merge any minutes.
     
-* **Release-note** Release note enforcement is another feather we are seeking from Mungegithub. With template, when prs are create, people should add release note (can be "None") in the pr description. Depended on the release message left, Mungegithub will add "release-note", "release-note-none". If you leave it empty, "release-note-needed" will be added and is going to block merging.
+* **Release-note** Release note enforcement is another feather we are seeking from Mungegithub. With template, when prs are create, people should add release note (can be the word "none") in the pr description. Depended on the release message left, Mungegithub will add "release-note-none"(if you write string `none`) or "release-note-action-required"(if you write string `action required`), "release-note"(Any other message). If you leave it empty, **"do-not-merge/release-note-label-needed"** will be added and is going to block merging.
 
-* **do-not-merge** Even with all required parts, you can always have more time by add "do-not-merge" label. 
+  > __New change after [PR#531](https://github.com/istio/test-infra/pull/531):__ _Mungegithub is no longer handle adding/removing "release-note" labels, instead, Prow takes care of that. And good news is after you added "release-note" in PR description, Prow will automatically remove "do-not-merge/release-note-label-needed" and unblock merge processes._
+
+* **do-not-merge** Merge process will be blocked in any stage due to the existence of any kinds of "do-not-merge" labels.
+
+  * _"do-not-merge"_: normal label to stop merge, can be added for general reason by people with write access.
+  * _"do-not-merge/hold"_: It's a easy for everyone to hold merge, simply comment **"/hold"** or **"/hold cancel"**, robot will add or remove "do-not-merge/hold" and it blocks/unblocks merge process.
```

Comment:
It's an easy way for anybody to hold the merge; if you simply comment, the robot will add and will block... use same verb tense

How does it unblock?

Reply:
By comment "/hold cancel", Prow will remove that label and the PR is good to go.